### ISSUE TITLE
When auto-restore fails due to unavailable media, offer an action for trying again

### ISSUE THREAD

**Issue by t-m-w** [stevesoltys/backup#768]
This is a minor convenience feature for a rainy day or something.

Current notification for failed auto-restore when a USB drive is missing suggests that the app must be uninstalled and reinstalled for its data to be restored. There is a notification action to Uninstall the app. However, given Seedvault is capable of restoring apps and their data at will nowadays, and selectively, it's capable of restoring the data later. Perhaps the notification could have an action to "Try Again" so that the user can plug in the USB drive and try to restore again, this time initiated directly by Seedvault.

Obscure use case here, except for a dev or tester, but if the app is installed via `adb install` - which often means connected to a computer via USB (although wireless debugging exists) - then you can't really have the USB drive plugged in at the same time. Workaround of course is to push the APK to the device and install it from there when the drive is plugged in - or, as mentioned, wireless debugging.

As said, this is very minor, since the user can manually restore the app's user data anyway from the menu.

**Comment by grote** [comment]
Good idea. We could even drop the `android.permission.REQUEST_DELETE_PACKAGES` permission in this case.
